corehq/apps/userreports/sql/connection.py

```diff
@@ -0,0 +1,65 @@
+import sqlalchemy
+from django.conf import settings
+
+
+DEFAULT_ENGINE_ID = 'default'
+
+
+def create_engine():
+    return sqlalchemy.create_engine(settings.SQL_REPORTING_DATABASE_URL)
+
+
+class ConnectionManager(object):
```

Comment:
Yes, it definitely makes sense. We only need one engine object per database. See docs for details: http://docs.sqlalchemy.org/en/rel_1_0/orm/session_basics.html